Description: Typo in test_dlls.py
 The following test has a typo, which makes it fail, and which I
 thereby fix.
Author: Pietro Battiston <toobaz@email.it>
Forwarded: yes

--- python-shapely-1.2.4.orig/shapely/tests/test_dlls.py
+++ python-shapely-1.2.4/shapely/tests/test_dlls.py
@@ -8,7 +8,7 @@ class LoadingTestCase(unittest.TestCase)
     def test_load(self):
         self.assertRaises(OSError, load_dll, 'geosh_c')
     def test_fallbacks(self):
-        a = load_dll('geosh_c', fallbacks=['/opt/local/lib/libgeos_c.dylib'])
+        a = load_dll('geos_c', fallbacks=['/opt/local/lib/libgeos_c.dylib'])
 
 def test_suite():
     return unittest.TestLoader().loadTestsFromTestCase(LoadingTestCase)
